pandas/io/tests/test_sql.py

```diff
@@ -682,6 +683,18 @@ def test_date_and_index(self):
         self.assertTrue(issubclass(df.IntDateCol.dtype.type, np.datetime64),
                         "IntDateCol loaded with incorrect type")
 
+    def test_sqlite_time(self):
+        # test support for datetime.time
+        df = DataFrame([time(9, 0, 0), time(9, 1, 30)], columns=["a"])
+        # GH 8341
+        # test if time object are correctly converted into string
+        # in a uniform way between sqlalchemy and the sqlite3 fallback
+        if self.flavor == 'sqlite':
+            sql.to_sql(df, "test_time", self.conn, index=False)
+            res = sql.read_sql_query("SELECT * FROM test_time", self.conn)
+            ref = df.applymap(lambda _: _.strftime("%H:%M:%S.%f"))
```

Comment:
yes, we decided to limit the PR on the writing part. We can an issue open for the reading